README.md

```diff
@@ -202,6 +195,9 @@ The build and installation instructions are idential for Ubuntu 16.04 and OS X.
 
 See the instructions provided in the [diagnostics] directory.
 
+https://github.com/NervanaSystems/ngraph-tf/blob/master/diagnostics/README.md
```

Comment:
@avijit-nervana 
instead of Add `<path-to-tensorflow-repo>/bazel-out/darwin-py3-opt/bin/tensorflow` and `<path-to-ngraph-tf-repo>/build/ngraph/ngraph_dist/lib` to your `DYLD_LIBRARY_PATH`
can we have the actual command here (so that its easy to copy paste):

export DYLD_LIBRARY_PATH=<path-to-tensorflow-repo>/bazel-out/darwin-py3-opt/bin/tensorflow:$DYLD_LIBRARY_PATH
export DYLD_LIBRARY_PATH=<path-to-ngraph-tf-repo>/build/ngraph/ngraph_dist/lib:$DYLD_LIBRARY_PATH

Reply:
Good catch.  Updated.